app/assets/stylesheets/components/common/_footer_secondary.scss

```diff
@@ -103,8 +104,33 @@
   }
 }
 
+.footer-secondary__sfgb-text {
+  display: inline-block;
+  & > * {
+    display: block;
+  }
+}
+
+.footer-secondary__sfgb-link {
+  margin-top: 0.75rem;
```

Comment:
Could we write this as a factor of $baseline-unit (*2 specifically in this case)?

Reply:
thanks @davidtrussler - updated this now